drivers/gpio/gpio_mcux_lpc.c

```diff
@@ -51,8 +51,13 @@ struct gpio_mcux_lpc_data {
 	sys_slist_t callbacks;
 	/* pin association with PINT id */
 	pint_pin_int_t pint_id[32];
+#ifdef CONFIG_SOC_SERIES_LPC51U68
 	/* ISR allocated in device tree to this port */
-	uint32_t isr_list[8];
+	uint32_t isr_list[4];
```

Comment:
Instead of having conditional code based on the platform, there is a define in the SOC header file for the count here that would be cleaner:

INPUTMUX_PINTSEL_COUNT

Reply:
@dleach02 fixed.